falcon/request.py

```diff
@@ -683,6 +687,22 @@ def access_route(self):
     def remote_addr(self):
         return self.env.get('REMOTE_ADDR')
 
+    @property
+    def port(self):
+        try:
+            host_header = self.env['HTTP_HOST']
+            host, port = parse_host(host_header)
+        except KeyError:
+            port = self.env.get('SERVER_PORT')
```

Comment:
SERVER_PORT will always be present, so `self.env.get('SERVER_PORT')` ===> `self.env['SERVER_PORT']`